### ISSUE TITLE
PW: Duplicate IDs on accordions after component update

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#14747]
## Describe the defect
There was a DST update to the <va-accordion> component done recently to allow for deep linking. Randi and PW had been working on this and these are the related tickets:
- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/11354
- https://github.com/department-of-veterans-affairs/content-build/pull/1650
- https://github.com/department-of-veterans-affairs/component-library/pull/782

However, after this update I now have duplicate IDs on accordions on quite a few pages.
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/de88778f-4c22-4018-b7cb-8126e5453480)

This is on both Facilities pages and PW pages:

## Example pages
**Pages that have the dup ID issue**
- https://www.va.gov/health-care/about-va-health-benefits/
- https://www.va.gov/resources/the-pact-act-and-your-va-benefits
- [Facilities] https://www.va.gov/south-texas-health-care/locations/north-central-federal-va-clinic/

**Pages that have accordions but do NOT have the issue**
- [Facilities] https://www.va.gov/boston-vet-center/
- [Facilities]  https://www.va.gov/salt-lake-city-health-care/programs/horses-helping-veterans
- https://www.va.gov/health-care/get-health-id-card/

## To Reproduce
1. Go to one of the pages above that I indicated has the issue
2. Scroll down to where there are accordions
3. Open the inspector and inspect an accordion element
4. Note that the `<va-accordion>` element and the `<h3>` element have the same ID

## Engineering notes

Cause is not known fully:
- [Slack thread with some initial discussion](https://dsva.slack.com/archives/C03LFSPGV16/p1691690032275419)
- In some cases it's not clear where the HC's ID is coming from
- Investigation should proceed template-by-template

Idea 1: remove ID from h3

### Sprint 91 ACs - 5 pt est.
- [x] Found list of templates in PW portfolio that have accordion components 
  - [x] Start with Josh M's [accordion audit](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/10380#issuecomment-1242022957) from some months ago 
  - [x] Compare to [this list in slack](https://dsva.slack.com/archives/C52CL1PKQ/p1691693309205649?thread_ts=1691687413.983129&cid=C52CL1PKQ)
- [x] Determined the cause for duplicate IDs in each template
- [x] Recommend approach to resolving each, discuss with Laura and team

### Future ACs
- [x] In Public Websites the duplicate ID is resolved (ID is changed or one is removed)
- [x] If we determine this is a global issue and that Facilities will be fixed by our changes, update #14748 to let Facilities know

## Screenshots
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/835fa519-775e-411d-9f1b-a93a92d9f536)

**Comment by laflannery** [comment]
[Facilities issue](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14748)

**Comment by randimays** [comment]
Waiting for prod deploy to validate and close.

**Comment by randimays** [comment]
Successfully validated in production. In all screenshots, note the ID assigned to the `va-accordion-item`. That ID should not be present on the `h4` or `h3` with the `slot="headline"`. I also checked a facilities page 👍 

<img width="1267" alt="Screenshot 2023-08-16 at 3 55 44 PM" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/19175324/01b98687-9b0c-4e3f-94be-372ddbcdf77f">
<br/>
<br/>
<br/>
<br/>
<img width="1221" alt="Screenshot 2023-08-16 at 3 54 25 PM" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/19175324/006a36f2-e8ee-41b9-9234-86a4a45df06a">
<br/>
<br/>
<br/>
<br/>
<img width="1226" alt="Screenshot 2023-08-16 at 3 57 30 PM" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/19175324/951a83e3-a0e0-43f4-a4db-abbdeeed2c82">

Closing!

**Comment by jilladams** [comment]
https://github.com/department-of-veterans-affairs/content-build/pull/1669
https://dsva.slack.com/archives/CDHBKAL9W/p1692292106699339

Reopening, this fix broke On This Page section / components on Resources & Support pages

**Comment by randimays** [comment]
Update: [PR posted](https://github.com/department-of-veterans-affairs/content-build/pull/1674) yesterday for review. Waiting for Facilities to take a look.

**Comment by laflannery** [comment]
Verified in prod, confirmed IDs are gone from the accordions and the On the page links are still present and functioning

**Comment by randimays** [comment]
Thanks @laflannery! I also validated in prod. Closing!
